code/modules/mob/living/carbon/human/human_defense.dm

```diff
@@ -617,14 +617,17 @@ emp_act
 	visible_message("<span class='danger'>[I] embeds itself in [src]'s [L.name]!</span>","<span class='userdanger'>[I] embeds itself in your [L.name]!</span>")
 	return TRUE
 
-/mob/living/carbon/human/proc/make_bloody_hands(list/blood_dna, b_color)
+/mob/living/carbon/human/proc/make_bloody_hands(list/blood_dna, b_color, amount=null)
 	if(isnull(b_color))
 		b_color = "#A10808"
 	if(gloves)
-		gloves.add_blood(blood_dna, blood_color)
+		gloves.add_blood(blood_dna, blood_color, amount)
 	else
 		hand_blood_color = b_color
-		bloody_hands = rand(2, 4)
+		if(isnull(amount))
+			bloody_hands = rand(2, 4)
+		else
+			bloody_hands = max(1, amount)
```

Comment:
Shouldn't this take into account how bloody the hands already are? It feels like we might be throwing the value away here/

Reply:
i dont like the idea of bloody hands stacking, if you would beat someone with bare hands while they are bleeding and we would just add value here, in the end you will go away with like 50 stacks, making everything bloody, it would look more like a bug than an intended mechanic. 
You get your hands bloody, you dont fill IV bags with the blood to spread on all the objects around, im not making this mistake again

Reply:
Fair enough. I'd still think it might be better to have this max value be a define or something, since this feels like an awfully magic number.

Reply:
done